frontend/src/app/utils/dates/dates.helper.ts

```diff
@@ -1,3 +1,7 @@
-export const isBetween = (date: Date, start: Date, end: Date) => {
-    return start <= date && date <= end;
-};
\ No newline at end of file
+export const intercept = (first: { start: Date, end: Date }, second: { start: Date, end: Date }) => {
+    return first.end >= second.start && second.end >= first.start;
+};
+
+export const hoursInDay = 24;
```

Comment:
There are different code styles for TS. Doesn't matter which one you are using (if any), but it would be better to name constants in the way you have understanding immediately what is it. E.g. CONSTANT_CASE.
https://google.github.io/styleguide/tsguide.html

Reply:
Fixed